### ISSUE TITLE
All monsters of the same type share the same stats

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#251]
```
Confirmed using v0.6.10a3.

All monsters of the same type re-use the same object for their stats (the 
CombatTraits), which means that if one monster's stats are changed, all other 
monsters of that type also changes stats.

This has not been a problem before, since you were unable to change monster's 
stats. However, with v0.6.10, we introduced items that can cause conditions on 
monsters, which makes this a problem. For example, the chaosreaper causes a 
condition that due to this issue affects all monsters of the same type once 
applied.

Instead, the monsters should get their own copy of the CombatTraits object. 
This would also need to be stored as a unique object per monster in the 
savegame file.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 12 Oct 2011 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved in r177.
```

Original comment by `oskar.wi...@gmail.com` on 12 Oct 2011 at 10:06
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
